app/assets/stylesheets/application.scss

```diff
@@ -122,6 +122,18 @@ $govuk-breakpoints: (
   }
 }
 
+.hackney-cautionary-contact {
+  color: white !important;
+}
+
+.hackney-cautionary-contact-table {
+  background-color: hackney-colour("hackney-warning");
```

Comment:
Prefixing with `hackney` here is redundant. `hackney-colour("hackney-warning")` has too much hackney in it.